scripts/qiita

```diff
@@ -289,6 +294,49 @@ def update_preprocessed_data(sl_out_dir, study,  preprocessed_data):
                                             preprocessed_data)
     click.echo("Preprocessed data %s successfully updated" % ppd.id)
 
+# #############################################################################
+# PORTAL COMMANDS
+# #############################################################################
+
+
+@portal.command(name="add-analyses")
+@click.option('--portal', required=True, type=str,
+              help='Portal to add the analyses to')
+@click.option('--analyses', required=True, type=str,
```

Comment:
I can, but that seems messy to me. If we have 30 analyses/studies to add, one comma list seems better to me than "-a 1 -a 2 -a 3 -a 4 -a 5 -a 6 -a 7 -a 8 ....."